app/charges/controllers.py

```diff
@@ -100,19 +96,20 @@ def get_charges(user, user_data, broadcast = False):
 @ensure_dict
 @get_user
 def get_charge(user, user_data, broadcast = False):
-
-    charge = Charges.query.filter_by(id= user_data["charge"]).first()
+    charge_id = user_data.get("charge", -1)
+    charge = Charges.query.filter_by(id= charge_id).first()
 
     if charge is None:
         emit('get_charge', Response.UsrChargeDontExist)
-        return;
+        return
 
     committee = Committees.query.filter_by(id = charge.committee).first()
     membership = committee.members.filter_by(member= user).first()
 
-    if charge.private and (membership is None and not user.is_admin):
```

Comment:
User wasn't checked before this, so it was possible to have user be None and cause an error.